bundles/org.openhab.binding.avmfritz/README.md

```diff
@@ -67,6 +67,11 @@ The FRITZ!DECT 400 supports a configurable button to trigger short or long press
 Beside four customizable buttons the FRITZ!DECT 440 supports temperature readings.
 ** NOTE: ** FRITZ!DECT 440 now uses Channel Groups to group its Channels like `device#battery_level`, `device#battery_low` for device information, `sensors#temperature` for sensor data and `top-left`, `bottom-left`, `top-right` and `bottom-right` combined with `press` and `last_change` (see [Full Example](#full-example))
 
+### FRITZ!DECT 500
+
+The [FRITZ!DECT 500](https://avm.de/produkte/fritzdect/fritzdect-500/) is a dimmable colorized light bulb.
```

Comment:
What about [Magenta SmartHome LED-Lampe farbig](https://www.smarthome.de/geraete/smarthome-led-lampe-e27-farbig-weiss)?

Reply:
i don't have one, so testing is hard to accomplish.

Reply:
i have both magenta variants (white + color) and willing to test